src/Routes/Abstract_REST_Route.php

```diff
@@ -0,0 +1,49 @@
+<?php // phpcs:disable WordPress.Files.FileName.NotHyphenatedLowercase
```

Comment:
I think we should actually name this `Abstract_Rest_Route` and make this work on case sensitive file systems: https://github.com/moderntribe/square-one/pull/678/files#r606454579